extensions/wikia/ARecoveryEngine/js/sourcePoint.js

```diff
@@ -43,20 +52,25 @@ define('ext.wikia.aRecoveryEngine.recovery.sourcePointHelper', [
 	 *
 	 * @returns {boolean}
 	 */
-	function isSourcePointRecoveryEnabled() {
+	function isEnabled() {
 		var enabled = !!context.opts.sourcePointRecovery && !context.opts.pageFairRecovery;
 
-		log(['isSourcePointRecoveryEnabled', enabled, 'debug', logGroup]);
+		log(['isEnabled', enabled, log.levels.debug, logGroup]);
 		return enabled;
 	}
 
 	function isBlocking() {
-		log(['isBlocking', !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking)], 'debug', logGroup);
-		return isSourcePointRecoveryEnabled() && !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking);
+		var isBlocking = !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking);
```

Comment:
yes, we sometimes want to check if user is blocking even though recovery is enabled